src/photon/photon_scheduler.c

```diff
@@ -346,14 +356,23 @@ int heartbeat_handler(event_loop *loop, timer_id id, void *context) {
   return LOCAL_SCHEDULER_HEARTBEAT_TIMEOUT_MILLISECONDS;
 }
 
+void start_new_worker(local_scheduler_state *state) {
+  /* We can't start a worker if we don't have the path to the worker script. */
+  CHECK(state->start_worker_command != NULL);
+  /* Launch the process to create the worker. */
+  FILE *p = popen(state->start_worker_command, "r");
```

Comment:
would be great to keep worker handles. I think we might want to both create and destroy workers in the local scheduler.